architecture.md

```diff
@@ -0,0 +1,16 @@
+# Architecture for CI
+
+Repo on GitHub
+    -Dev local test and setup
+    
+    GH Actions to test and format code on PR
+        - run tests
+        - Run format, prettier
+        - scan code for "bad" words
+        - scan code for single ' quotes. 
+    
+    Build container on successful merge with main
+        - installs dependencies
+        - runs node
+        - npm command starts app 
+    Deploy container on Digital Ocean
```

Comment:
It isn't really doing all of these things though, is it? Or do you have scripts elsewhere (e.g. DigitalOcean) that do these things when the repo on GitHub changes?